src/main/java/knightminer/simplytea/data/gen/RecipeGenerator.java

```diff
@@ -116,10 +111,10 @@ protected void buildCraftingRecipes(Consumer<FinishedRecipe> consumer) {
 
 		// advanced tea
 		addTea(consumer, cup_cocoa, Items.COCOA_BEANS, Items.COCOA_BEANS, teapot_frothed);
-		addHoney(consumer, cup_cocoa, tea_stick, CocoaItem.CINNAMON_TAG);
+		addHoney(consumer, RecipeCategory.MISC, cup_cocoa, tea_stick, CocoaItem.CINNAMON_TAG);
```

Comment:
I feel like most of the tea crafting recipes should be under food.

The pots themselves I think misc is reasonable, I could go either way on the tea bags and alike. But anything that is edible I'd put under food.